### ISSUE TITLE
[BUG] First call to CREATE OR REPLACE TABLE shows confusing INFO

### ISSUE THREAD

**Issue by randerzander** [dask-contrib/dask-sql#391]
If the first table a user creates is created with `CREATE OR REPLACE` syntax, they get a confusing INFO:

```
CREATE OR REPLACE TABLE stations WITH (
    location = '/raid/weather_meta/stations.parquet',
    gpu=True
)

# in Jupyter, red INFO line:
No tables are registered.
```

The table is successfully created, but the INFO message makes it seem like it might not have been.

Is there a reason to emit this message at all?

**Comment by charlesbluca** [comment]
No, I'm not sure why we choose to log that message at the `warning` level, as there are various cases where one might perform SQL queries without actually registering any tables - I can start off a PR switching this to a debug level log, and also look through the code to see if there's any other places where it might be more fitting to switch the log level.
